packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -176,7 +176,7 @@ class Auth {
     constructor(config: AuthConfig) {
         // Special endpoint for injecting SLAS private client secret
         const baseUrl = config.proxy.split(`/mobify/proxy/api`)[0]
```

Comment:
I think there is a bug in the waiting here as well. Since the proxy path value is configurable we might run into issues if someone was to change that.